_posts/blog/2018-10-17-ReviewGarcíaMárqu

```diff
@@ -0,0 +1,63 @@
+
+Gracias a la publicación de este archivo podemos entrar en la casa del premio Nobel y hacernos amigo de él o no...
+
+
+...En la colección digital, no sólo se puede buscar por palabras claves sino que se puede comparar las múltiples versiones del escritor sobre una misma novela. De esa forma podemos ver como el García Márquez va re-versionando su propio trabajo hasta el producto final, por ejemplo...
+
+
+...El archivo digital también permite conocer la vida del escritor en dos idiomas, ingles y español. Le da acceso a todas las personas que tengan internet y abre un mundo de posibilidades para los estudios sobre el escritor, sobre la época que vivió, sobre México donde vivió durante muchos años, sobre el origen de su musa y el origen de sus obras y todo lo que un archivo en sí puede ofrecer.
+
+Sin embargo, no se trata nada más sobre lo que el archivo pueda ofrecer, o el hecho de que está digitalizado y es accesible a todo el mundo. Tampoco se trata nada más, por ejemplo, sobre el hecho de que la colección privada de un escritor latinoamericano se encuentre en otro país lejos de la tierra donde él vivió...
+
+
+...El caso del archivo de Gabriel García Márquez también nos abre a la reflexión sobre lo que significa un archivo per se debido a la polémica que se generó cuando la familia decidió vendérselo al Harry Ransom Center.
+
+Entonces vamos más allá del archivo. Más allá de una simple recolección de papeles depositados en 24 cajas. El archivo como bien lo describían los autores Carlos Acuña y Javier Villa Flores en el texto “Los archivos y la construcción de la verdad histórica en América Latina” son:	
+		
+ *“Lejos de ser depósitos imparciales, los archivos operan a partir de relaciones de autoridad y fuerza que determinan lo que es “archivable” y lo que es “accesible”. Su propia creación, contenido y funcionamiento son el resultado de acciones deliberadas de inclusión y exclusión tendientes a “organizar”, “ordenar” y, por lo mismo, construir” el pasado.”*...
+ 
+
+...El archivo significa un pasado, un presente y un futuro. Y en este sentido es comprensible que el traslado, por ejemplo, de estos archivos a otro país haya generado tanta polémica...
+
+
+...Cuando el escritor murió en el 2014 y se dió a conocer la noticia del destino de sus archivos, muchas fueron las reacciones a favor y en contra de la decisión. La idea de que el escritor les pertenece a los colombianos se diluyó al vender su memorabilia a un centro en los Estados Unidos. Yo diría que con no sólo con esta decisión García Márquez dejó de ser de los colombianos, sino que una vez que se digitalizó el archivo, se perdió en el mundo global del internet. Ahora el escritor está en la nube para todo todo el mundo. Todos los que tiene acceso a internet pueden gozar de la presencia de él en su espacio.
+
+Como consecuencia, el archivo se convirtió en algo etéreo y concreto a la vez...
```

Comment:
The archive might feel ethereal, but that's not entirely true. For example: there are legal restrictions that explain why some of the documents were digitized, while others were not. Similarly, the digital files have a material impact (they are literally inscribed on discs) and are embedded in an equally material repository software and metadata schema. I'd like you to think more about the reality of the archive.